index.yaml

```diff
@@ -33,6 +33,17 @@ indexes:
   - name: commit_message
   - name: commit_type
 
+- kind: DeleteExpCopiesRealtimeModel
```

Comment:
Just wondering -- why did this file change?

Reply:
I'm actually not completely sure why my PRs make the index.yaml change, because I haven't touched the startup script at all. I'll revert them.